client/blipsnames.lua

```diff
@@ -38,6 +38,9 @@ end)
 RegisterNetEvent('qb-admin:client:Show', function(players)
     for _, player in pairs(players) do
         local playeridx = GetPlayerFromServerId(player.id)
+        if playeridx == -1 and player.id ~= PlayerId() then
+            return
```

Comment:
note that returning here will stop the for loop. probably best to add a goto instead

Reply:
Yeah you're right I'll fix it in a min